docs/smart-contract-backend/scb.md

```diff
@@ -69,6 +69,38 @@ The node client is a process that talks to a Cardano node. It can submit transac
 2. Ask for all unspent outputs at an address (see 1.2.1.5)
 3. Ask for the current slot number (see 1.2.1.1)
 
+We communicate with the node client using a REST API, and the listener will establish a web sockets connection and be updated regularly with accepted transactions.
+
+The mock will implement the following endpoints (which will also be exposed by the real node):
+
+```
+--| Notations
+    <-   Client sends to Server.
+    ->   Server sends to Client.
+    <d>  Datum `d` is JSON encoded.
+
+data Update = Rollback Slot <Block>
+            | NewBlock Slot <Block>
+
+-- API
+GET /utxo/:addr
+-> <[(TxOutRef, TxOut)]>
+
+GET /currentSlot
```

Comment:
What purpose does this endpoint serve?

Reply:
It gets the utxo at an address.

Reply:
Do you have a link to a specification describing this protocol?

Reply:
@rhyslbw The `/currentSlot` gets the current slot number. In a real system we don't need to ask the node for this, but on the mockchain the slots are not aligned to actual time spans so we need to ask for the slot number

Reply:
@raduom I'm not sure if it's documented, but here is a "template" showing an implementation: https://github.com/input-output-hk/cardano-node/blob/5b0339a8e06175b2945706222e060163490526c1/cardano-node/src/Cardano/Wallet/Client.hs

@j-mueller 
> but on the mockchain the slots are not aligned to actual time spans so we need to ask for the slot number

Who is "we" in "we need to ask for the slot number"? Having the latest block gives the subscriber this information.